node-attestation-bindings/Cargo.toml

```diff
@@ -14,7 +14,7 @@ crate-type = ["cdylib"]
 # Default enable napi4 feature, see https://nodejs.org/api/n-api.html#node-api-version-matrix
 napi = { version = "2.10.6", default-features = false, features = ["napi4"] }
 napi-derive = "2.9.4"
-attestation-doc-validation = "0.5.0-beta"
+attestation-doc-validation = "0.6.0"
```

Comment:
Should these be major version bumps as the param has changed type?

Reply:
As they're still on major version 0, which is [considered unstable](https://semver.org/#spec-item-4), a major version isn't needed for breaking changes. You can see in the breaking [changes recommendation](https://semver.org/#spec-item-8) that it also says it's only required for major versions > 0